cmpserver/plugin/plugin.go

```diff
@@ -357,7 +357,7 @@ func (s *Service) matchRepository(ctx context.Context, workdir string, envEntrie
 	if len(config.Spec.Discover.Find.Command.Command) > 0 {
 		log.Debugf("Going to try runCommand.")
 		env := append(os.Environ(), environ(envEntries)...)
-		find, err := runCommand(ctx, config.Spec.Discover.Find.Command, appPath, env)
+		find, err := runCommand(ctx, config.Spec.Discover.Find.Command, false, appPath, env)
```

Comment:
I think `true` also makes sense here, since zero-output means "CMP doesn't support your repo".

Reply:
In our case, that's the case 95% of the time...